modules/package.json

```diff
@@ -13,7 +13,10 @@
 	"private": true,
 	"resolutions": {
 		"@liferay/eslint-plugin": "1.2.0",
-		"ua-parser-js": "^0.7.30"
+		"@types/html-minifier-terser": "5.1.0",
+		"html-webpack-plugin": "5.5.0",
+		"ua-parser-js": "^0.7.30",
+		"webpack": "5.70.0"
```

Comment:
Is there a point when we wouldn't need these resolutions? Or are we dependent on the maintainers of these packages to correctly specify the versions we need?

Reply:
TBH, I have no idea. I just did this to unblock the progress, but we need to carefully examine what's going on and how it should be fixed.